src/main/java/frc/robot/utils/SRXMagEncoder.java

```diff
@@ -18,20 +16,21 @@ public SRXMagEncoder(DutyCycle cycle, double offset) {
     super(cycle);
     this.offset = offset;
     setDistancePerRotation(360);
+    setDutyCycleRange(offset, offset);
```

Comment:
I think this would need to be `1` and `4096` but I'm not exactly confident. Could you revert this line & we can test the mag encoders with the green LEDs in the pits before messing with the duty cycle range? I'm fine with the rest of the formatting changes to this file since they don't affect functionality (cleaner code = good even though it isn't related to the PR goal!)